<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxulke">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>ülkeler</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>381</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutulke">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonturkiye">
        <property name="text">
         <string>türkiye</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonazerbeycan">
        <property name="text">
         <string>azerbeycan</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonalmanya">
        <property name="text">
         <string>almanya</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxegitm">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>eğitim</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutegitim">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonlise">
        <property name="text">
         <string>lise</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonyukseklisans">
        <property name="text">
         <string>yüksek lisans</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonuniversite">
        <property name="text">
         <string>üniversite</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonulke">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>ülke seçimi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonegitim">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>eğitim seçimi</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelulke">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelegitim">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
